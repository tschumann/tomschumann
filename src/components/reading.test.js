import React from 'react';
import renderer from 'react-test-renderer';
import Reading from './reading';

test('test component renders', () => {
	const component = renderer.create(<Reading title="My site" link="http://dev.lan" key="1"/>);
	const tree = component.toJSON();

	expect(tree).toMatchSnapshot();
});

