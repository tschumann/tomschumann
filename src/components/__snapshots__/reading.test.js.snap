// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component renders 1`] = `
<div>
  <h3
    className="book-title"
  >
    My site
  </h3>
   
  <a
    href="http://dev.lan"
    target="_blank"
  >
    http://dev.lan
  </a>
</div>
`;
